docs/microbit_micropython_api.rst

```diff
@@ -102,9 +136,26 @@ Each of these pins are instances of the ``MicroBitPin`` class, which offers the
     # sets the period of the PWM output of the pin in microseconds
     # (see https://en.wikipedia.org/wiki/Pulse-width_modulation)
     pin.set_analog_period_microseconds(int)
-    # returns boolean
+    # returns boolean for touch pins; 0,1 and 2
```

Comment:
Was this meant to be a semicolon?

Reply:
It was. Assuming that  `;` indicates a list is to follow. Not sure if it's really needed

Reply:
Can it also be more explicit about the method is only available in those pins?

Reply:
> It was. Assuming that ; indicates a list is to follow. Not sure if it's really needed

I would maybe then suggest something like `Only available for touch pins 0, 1, and 2. Returns boolean if the pin is touched.` ?

Reply:
Thanks. Have updated accordingly